dist/ansible/ovn-playbook.yaml

```diff
@@ -79,6 +79,9 @@
     shell: oc adm policy add-scc-to-user anyuid -z ovn
     ignore_errors: yes
 
+  - name: Patch node to run on Linux only (there are Windows nodes as well)
+    shell: oc patch namespace openshift-node -p '{"metadata":{"annotations":{"openshift.io/node-selector":"beta.kubernetes.io/os=linux"}}}'
```

Comment:
The daemonset spec should have the correct selector already. Why do we need this on top of that?

Reply:
Glenn ran into a problem trying to start on the windows nodes

Reply:
@pecameron Can we fix this here if at all?
https://github.com/openvswitch/ovn-kubernetes/blob/master/dist/yaml/ovn-namespace.yaml#L14
That is where the namespace is born.

Reply:
Good spot. I made the change.